src/index.html

```diff
@@ -1,14 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>2048</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
     <div class="container">
-      <div class="game-header">
+      <div class="game_header">
```

Comment:
Class names should be written in lowercase with hyphens separating words (kebab-case). Instead of 'game_header' use 'game-header'.
Fix all similar cases